### PR TITLE
feat: IOC-7354 - added new API to get org services v2

### PR DESCRIPTION
**Pull request by dc297**
## Description

1. Updated api.json to latest from transporter's swagger
2. Updated README and other documentation
3. Added new API for fetching org services v2. This new API was added so we can fetch services available to an org-user combination irrespective of the user's role in the org. The existing version only works for `developer` or `System Admin` role
## Related Issue

https://jira.corp.adobe.com/browse/IOC-7354
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-lib-console",
-  "version": "5.2.0",
+  "version": "5.2.1",
```
Comment (MichaelGoberling) [inline package.json:3]: Revert - this will get updated automatically when we release

**Review by MichaelGoberling — APPROVED** [review]
LGTM besides reverting package.json

File: package.json (hunk 1 above)
Comment (dc297) [reply package.json]: Done, reverted this. Also added tests and a bug fix.

**Review by arjuncooliitr — APPROVED** [review]
LGTM

**Pull request merged**